G_channels.md

```diff
@@ -249,6 +249,81 @@ socket = assign(socket, :user, msg["user"])
 
 Sockets store assigned values as a map in `socket.assigns`.
 
+#### Using Token Authentication
+
+When we connect, we'll often need to authenticate the client. Fortunately, this is a fairly easy 4-step process with `Phoenix.token`. 
+
+Step 1 - Assign Token in Connection:
+
+For example, let's assume that we have an authentication plug, `OurAuth`, in our app which has already been used to authenticate the user's session and assign a `:current_user`. Since the current user exists we can simply assign the user's token in the connection for use in the layout. To do so, in our "web/router.ex": we'll need to add the `OurAuth` plug and make the assignment, as follows:
```

Comment:
I took a stab at another version of this paragraph. Please let me know what you think.

Let's say we have an authentication plug in our app called `OurAuth`. When OurAuth authenticates a user, it sets a value for the `:current_user` key in `conn.assigns`. Since the current user exists, we can simply assign the user's token in the connection for use in the layout. We can wrap that behavior up in a private function plug, `put_user_token/2`. To make this all work, we just add `OurAuth` and `put_user_token/2` to the browser pipeline.